.github/workflows/font_tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Font tests
+on:
+  push:
+    paths:
+      - 'src/**'
+      - 'test/font/**'
+      - '.github/workflows/font_tests.yml'
```

Comment:
Here, and below, should we perhaps also include `gulpfile.js` and `test/test.mjs` as well since changes to either of those files *may* affect this test?